### ISSUE TITLE
Some cell values are not read - possibly related to formulas

### ISSUE THREAD

**Issue by jkyeung** [PydPiper/pylightxl#12]
pylightxl fails to read some of the cell values in the attached sheet:

[sample1.xlsx](https://github.com/PydPiper/pylightxl/files/4607974/sample1.xlsx)

In particular, C8, C9, and C10 are reported as empty when in fact they should all be `'G'`.

**Comment by PydPiper** [comment]
Hi jkyeung, which version of pylightxl were you using when you had this issue? 

The latest version 1.42 off pypi read in column C just fine on my end:

> import pylightxl as xl
> db = xl.readxl('sample1.xlsx')
> db.ws('Sheet1').col(3)
> ['G', ',2', 'F0', ',0', 'P0', 'P3', 'D4', 'G', 'G', 'G']

**Comment by jkyeung** [comment]
I tested with pylightxl 1.42, Python 3.6.3, Windows 10.

**Comment by PydPiper** [comment]
going to close this out. We were unable to replicate the issue

**Comment by jkyeung** [comment]
That's unfortunate. I just noticed that I actually don't even get cell C7:

```
Python 3.6.3 (v3.6.3:2c5fed8, Oct  3 2017, 18:11:49) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import pylightxl as xl
>>> db = xl.readxl('sample1.xlsx')
>>> db.ws('Sheet1').col(3)
['G', ',2', 'F0', ',0', 'P0', 'P3', '', '', '', '']
```

**Comment by PydPiper** [comment]
![image](https://user-images.githubusercontent.com/35475223/83096230-03d4cd00-a05a-11ea-94fe-0f9b93b358d1.png)

sorry we were unable to replicate the issue on python 3.6.3 win10 same version as you

**Comment by jkyeung** [comment]
It looks to me as though this was solved by the following from the 1.43 release notes:

> bug fix: accounted for edge case that was prematurely splitting cell tags by formula closing
> bracket

I guess your "1.42" included [this commit](https://github.com/PydPiper/pylightxl/commit/ade2de837129fe71d3682d9cde614e4cdfb86a12) and mine didn't.

I'm glad it's fixed.
